File: 2 fibonacci/FiboRecur.java
public class FiboRecur {
    static int counter = 2;

    public static void fibo(int first, int second) {
        if (counter <= 15) {
            int cur = first + second;
            System.out.println(cur);
            second = first;
            first = cur;
            counter += 1;
            fibo(first, second);
        } else {
            return;
        }
    }

    public static void main(String[] args) {
        System.out.println(0);
        System.out.println(1);

        fibo(1,0);
    }
}

File: 2 fibonacci/fiboRecur.py
print(0)
print(1)

counter = 2

def fibo(first, second):
    global counter
    if counter <= 15:
        cur = first + second
        print(cur)
        second = first
        first = cur
        counter += 1
        fibo(first, second)
    else:
        return

fibo(1,0)